src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/Internal/TimeSeriesFormCallout.tsx

```diff
@@ -94,6 +101,8 @@ const TimeSeriesFormCallout: React.FC<IProp> = ({
             setSeriesToEdit(
                 produce((draft) => {
                     draft.expression = newPropertyExpression.expression;
+                    draft.propertyType =
```

Comment:
when you write an expression how will we know the type?

Reply:
You can't type an expression on this one - unless I'm missing something there's no advanced mode due to it being adx